turbopack/crates/turbopack-core/src/module_graph/mod.rs

```diff
@@ -404,17 +430,25 @@ impl SingleModuleGraph {
         self.graph.node_references()
     }
 
-    pub fn read(self: &ReadRef<SingleModuleGraph>) -> ModuleGraphRef {
+    pub fn read(
+        self: &ReadRef<SingleModuleGraph>,
+        graph_idx: Option<u32>,
```

Comment:
```suggestion
        graph_idx: u32,
```

Not sure if I did understand it correctly, but don't you have to pass a graph_idx here.
Passing None is very likely incorrect, since the vec index probably doesn't match the graph idx